damnit/gui/table.py

```diff
@@ -10,9 +11,10 @@
 from PyQt5 import QtCore, QtWidgets, QtGui
 from PyQt5.QtCore import Qt
 
-
 ROW_HEIGHT = 30
 THUMBNAIL_SIZE = 35
+# The actual threshold for long messages is around 6200
+# not 10k, otherwise one gets a '414 URI Too Long' error
```

Comment:
This is strange, I'd never seen that and I've sent tons of messages over the api 🤔 
Do you see this by posting directly to zulip or through the proxy? and is it the same on zulip cloud and in our test instance?

Reply:
It's a proxy only issue. Directly sending with the zulip API has the usual threshold.

Reply:
OK we should check that with @RobertRosca when he's back then

Reply:
Had a look and the cause is likely Nginx's limit on URI length. I could bump that up, but really having the message content as a query parameter is kind of silly, so instead I'll make it a body parameter which will get past this issue.